controllers/appwrapper_controller.go

```diff
@@ -112,8 +120,10 @@ func (r *AppWrapperReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 	}
 
 	if !appwrapper.ObjectMeta.DeletionTimestamp.IsZero() || appwrapper.Status.State == arbv1.AppWrapperStateCompleted {
-		if err := r.finalizeScalingDownMachines(ctx, &appwrapper); err != nil {
-			return ctrl.Result{}, err
+		if !r.annotationExists(&appwrapper, "scaledownComplete") && len(getInstanceRequired(appwrapper.Labels)) > 0 {
```

Comment:
Is there a time when using instascale that the ordered instance label will be 0; Is the label removed from the app wrapper when the resource is deleted?

Reply:
Hey @Fiona-Waters 
Instascale currently only looks for the presence of an appwrapper to scale up, so if an appwrapper is deployed without the intent of using instascale, ie) no  orderedinstance, instascale will still recognise this and print `"Found AW <name> that cannot be scaled due to missing orderedinstance label`. But then upon deleting the appwrapper, instascale continues to print `Error deleting MachineSet for AppWrapper...` even though there has been no previous scale-up. 

Although in the sdk, there is an enable instascale field, selecting this only determines the ability to add instances for the generation of the appwrapper, but does not add any fields to the appwrapper stating that instascale is enabled, other than the instances. 

This can be seen by creating a generic aw such as:
```
apiVersion: workload.codeflare.dev/v1beta1
kind: AppWrapper
metadata:
  name: 0001-aw-generic-deployment-1
spec:
  resources:
    GenericItems:
    - replicas: 1
      generictemplate:
        apiVersion: apps/v1
        kind: Deployment
        metadata:
          name: 0001-aw-generic-deployment-1
          labels:
            app: 0001-aw-generic-deployment-1
        spec:
          selector:
            matchLabels:
              app: 0001-aw-generic-deployment-1
          replicas: 2
          template:
            metadata:
              labels:
                app: 0001-aw-generic-deployment-1
            spec:
              containers:
               - name: 0001-aw-generic-deployment-1
                 image: kicbase/echo-server:1.0
                 ports:
                 - containerPort: 80
                 resources:
                   requests:
                     cpu: 100m
                     memory: 256Mi
                   limits:
                     cpu: 100m
                     memory: 256Mi
```

Reply:
That makes sense, thanks for the explanation! :) 
/lgtm